### PR TITLE
New: Add `fetch::missing::<resource-type>` event

### PR DESCRIPTION
**Pull request by alrra**
Ref: #960

### CODE REVIEW

File: packages/sonarwhal/docs/contributor-guide/getting-started/events.md [hunk 1]
```diff
@@ -77,10 +78,23 @@ type FetchError {
...
+## `fetch::missing::<resource-type>`
+
+Event is emitted **when** a resource is not found.
+
+**Format:**
+
+```ts
+type FetchMissing {
+    /** The URL to download */
+    resource: string;
```
Comment (sarvaje) [inline packages/sonarwhal/docs/contributor-guide/getting-started/events.md]: When the resource is missing, I thing that most of the time we don't now the url/path

Comment (alrra) [reply packages/sonarwhal/docs/contributor-guide/getting-started/events.md]: I forgot to change this to `/** The URL of the target. */`. 

Also:
  * `resource` this should be optional (for cases that do not have it)? 
  * Maybe this should also include an `element` also optional for cases such as `<link rel="manifest" href="">`? Or should that be another case?

Comment (sarvaje) [reply packages/sonarwhal/docs/contributor-guide/getting-started/events.md]: > `resource` this should be optional (for cases that do not have it)?

I think so.

**Comment by alrra** [comment]
Talking a bit with @molant and @sarvaje, and we decided to see in the future how to better address this if we event have to (most of the time there will probably be just one rule that will use this event).

---

Note: I merged the improvements in https://github.com/sonarwhal/sonarwhal/commit/c62ad58e247af019ec1d6ea330b2317e963bdb08.

**Pull request closed without merging**